WcaOnRails/spec/views/registrations/psych_sheet_event.html.erb_spec.rb

```diff
@@ -23,7 +23,7 @@
     newcomer2 = FactoryGirl.create(:user, name: "Newcomer II")
 
     [the_best, tied_first, tied_second, newcomer1, newcomer2].each do |user|
-      FactoryGirl.create(:registration, :accepted, user: user, competition: competition, event_ids: ["333"])
+      FactoryGirl.create(:registration, :accepted, user: user, competition: competition, events: [Event.find("333")])
```

Comment:
There's no way it *really* worked. We used this handy `event_ids` before but it has been changed in [this commit](https://github.com/cubing/worldcubeassociation.org/commit/d3b2fe4a48372340124a24cc84507923d6758cb0) (see `WcaOnRails/spec/factories/registrations.rb` there). I've just checked and this is the only place that hasn't been updated =)